classes/TaskRunner/Upgrade/BackupDb.php

```diff
@@ -159,8 +159,8 @@ public function run()
                 $schema = $this->container->getDb()->executeS('SHOW CREATE TABLE `' . $table . '`', true, false);
 
                 if (count($schema) != 1 ||
-                    !((isset($schema[0]['Table']) && isset($schema[0]['Create Table']))
-                        || (isset($schema[0]['View']) && isset($schema[0]['Create View'])))) {
+                    !((isset($schema[0]['Table'], $schema[0]['Create Table']))
```

Comment:
You can also remove useless parenthesis :D 
(I know you hate me :roll_eyes: )

Reply:
Fixed ;) Thanks